code/modules/mob/living/carbon/xenomorph/castes/praetorian/abilities_praetorian.dm

```diff
@@ -220,7 +220,7 @@ GLOBAL_LIST_INIT(acid_spray_hit, typecacheof(list(/obj/structure/barricade, /obj
 	span_danger("We dash towards \the [A], spraying acid down our path!") )
 	succeed_activate()
 
-	owner.pass_flags = HOVERING
+	owner.pass_flags = PASS_LOW_STRUCTURE|PASS_DEFENSIVE_STRUCTURE|PASS_FIRE
```

Comment:
Why? Its meant to be able to spray multiple people in a path.

Reply:
the desc explicitly says otherwise, and currently it doesn't work at all. It just phases through mobs.